LINUX/configure

```diff
@@ -927,17 +927,21 @@ EOF
 Since the generic driver has been disabled, the corresponding NICs
 will NOT be usable by netmap.
 EOF
-				fi 
+				fi
 			} | warning
 			else
 				src=$(cd $src; pwd)
 				src_found=1
-			fi 
+			fi
 			src_checked=1
 		fi
 		drv_kernel_src=
 		if [ -n "$src_found" ]; then
-			drv_kernel_src=$(find $src/drivers/ -name "$d" -print -quit)
+			if [ "$d" = "stmmac_eth" ]; then
```

Comment:
We don't want to fill the script with special cases. Can't you just call the driver ```stmmac```?

Reply:
i think not possible since the Kconfig symbol for the driver is [ CONFIG_STMMAC_ETH ], while the driver directory is [ drivers/net/ethernet/stmicro/stmmac/ ]
and during configuration stage we get: following 
fashkar@barbarian:~/Downloads/EncDir/sperry/forked-netmap$ CONFIG_STMMAC=m CONFIG_E1000=m ./configure --driver-suffix=-netmap --no-ext-drivers --drivers=e1000,stmmac
...
********************************** WARNING **********************************
*** unknown driver: stmmac
*****************************************************************************

Reply:
OK, but you can add

```stmmac@conf := CONFIG_STMMAC_ETH```

to `LINUX/default-config.make.in_`. This will override the Kconfig symbol for just that driver.

Reply:
Ack

Reply:
ack